packages/admin/blocks-admin/src/blocks/factories/createBlocksBlock.tsx

```diff
@@ -588,13 +624,12 @@ export function createBlocksBlock({
                                                                     onDeleteClick={() => {
                                                                         deleteBlocks([data.key]);
                                                                     }}
-                                                                    moveBlock={(from, to) => {
-                                                                        updateState((prevState) => {
-                                                                            const blocks = [...prevState.blocks];
-                                                                            const blockToMove = blocks.splice(from, 1)[0];
-                                                                            blocks.splice(to, 0, blockToMove);
-                                                                            return { ...prevState, blocks };
-                                                                        });
+                                                                    moveBlock={(dragIndex: number, hoverIndex: number) => {
+                                                                        const blocks = [...state.blocks];
+                                                                        const dragItem = state.blocks[dragIndex];
+                                                                        blocks[dragIndex] = state.blocks[hoverIndex];
+                                                                        blocks[hoverIndex] = dragItem;
+                                                                        updateState((prevState) => ({ ...prevState, blocks }));
```

Comment:
Why are you doing this? Looks like you are reverting the fix you made in https://github.com/vivid-planet/comet/pull/1344/files?

Reply:
This wasn't intentional. Used an old local copy that was obviously started before the fix. Reverted: b05dc94095878cb7211e2b0c6df7dfeca7523be6